src/util/saveArticles.ts

```diff
@@ -868,33 +868,31 @@ async function templateArticle(parsoidDoc: DominoElement, moduleDependencies: an
     /* Set footer */
     const div = htmlTemplateDoc.createElement('div');
 
-    if (articleDetail.revisionId) {
-        /* Revision date */
-        const date = new Date(articleDetail.timestamp);
-        const lastEditedOnString = date ?
-            interpolateTranslationString(dump.strings.LAST_EDITED_ON, {
-                date: date.toISOString().substring(0, 10),
-            }) : null;
-
-        const creatorLink =
-            `<a class="external text" ` +
-            `${lastEditedOnString ? `title="${lastEditedOnString}"` : ''} ` +
-            `href="${mw.webUrl.href}?title=${encodeURIComponent(articleId)}&oldid=${articleDetail.revisionId}">` +
-            `${dump.mwMetaData.creator}</a>`;
-
-        const licenseLink =
-            `<a class="external text" href="https://creativecommons.org/licenses/by-sa/4.0/">${dump.strings.LICENSE_NAME}</a>`;
-
-        div.innerHTML = footerTemplate({
-            disclaimer: interpolateTranslationString(dump.strings.DISCLAIMER, {
-                creator: creatorLink,
-                license: licenseLink,
-            }),
-            strings: dump.strings,
-        });
-        htmlTemplateDoc.getElementById('mw-content-text').appendChild(div);
-        addNoIndexCommentToElement(div);
-    }
+    /* Revision date */
+    const date = new Date(articleDetail.timestamp);
+    const lastEditedOnString = date ?
+        interpolateTranslationString(dump.strings.LAST_EDITED_ON, {
+            date: date.toISOString().substring(0, 10),
+        }) : null;
+
+    const creatorLink =
+        `<a class="external text" ` +
+        `${lastEditedOnString ? `title="${lastEditedOnString}"` : ''} ` +
+        `href="${mw.webUrl.href}?title=${encodeURIComponent(articleId)}&oldid=${articleDetail.revisionId}">` +
```

Comment:
if revision Id is not present shouldn't we add a link to main wikipedia page......should this be done here as well?

Reply:
What is a scenario for not having a revisionId?